Lib/test/test_deque.py

```diff
@@ -949,6 +929,21 @@ def test_free_after_iterating(self):
         # For now, bypass tests that require slicing
         self.skipTest("Exhausted deque iterator doesn't free a deque")
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_contains_fake(self):
+        super().test_contains_fake()
+
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_count(self):
+        super().test_count()
+
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_index(self):
+        super().test_index()
```

Comment:
When I add things like this, I'd like to direct contributors against leaving the last two lines of this pattern behind when they un-mark these tests.